tests/std/tests/P0645R10_text_formatting_utf8/test.cpp

```diff
@@ -69,7 +69,7 @@ void test_width_estimation() {
         {TYPED_LITERAL(CharT, "\x58"), 1},
         {TYPED_LITERAL(CharT, "x\x58"), 2},
 
-        // test the boundaries of the intervals defined in n4885 [format.string.std]/11
+        // Test the boundaries of the intervals defined in n4885 [format.string.std]/11

```

Comment:
This comment is outdated and I think it doesn't match well with the following test cases; I have no idea how to change it though.

Reply:
We should consistently refer to `N4950 [format.string.std]/13`.

Reply:
Let's just phone it in and cite the new paragraph without further elaboration.